packages/cfd/src/Containers/cfd-dashboard.tsx

```diff
@@ -464,6 +466,7 @@ const CFDDashboard = (props: TCFDDashboardProps) => {
     const verification_code = platform === CFD_PLATFORMS.MT5 ? mt5_verification_code : dxtrade_verification_code;
 
     if (platform === CFD_PLATFORMS.DXTRADE && !is_dxtrade_allowed) return <Redirect to={routes.mt5} />;
+    if (platform === CFD_PLATFORMS.MT5 && is_pre_appstore) return <Redirect to={routes.traders_hub} />;
```

Comment:
we need same for derivx
```suggestion
    if ((platform === CFD_PLATFORMS.MT5 || platform === CFD_PLATFORMS.DXTRADE ) && is_pre_appstore) return <Redirect to={routes.traders_hub} />;
```

Reply:
@mahdiyeh-deriv we already had a condition to redirect derivx url to mt5 if derivx is not allowed. so added the new condition in the same block. pls help to check it

Reply:
you can change the hierarchy of the conditions :
```suggestion
if ((platform === CFD_PLATFORMS.MT5 || platform === CFD_PLATFORMS.DXTRADE ) && is_pre_appstore) return <Redirect to={routes.traders_hub} />;
if (!is_dxtrade_allowed) return <Redirect to={routes.mt5} />;

```